### DISCUSSION TITLE
Explore usability

### DISCUSSION THREAD

**Discussion by elibyy** [community/community#64623] [Mobile]
### Select Topic Area

Product Feedback

### Body

Hey there,
I really enjoy exploring but sometimes when I'm on the go (using mobile data) i get those pesky messages.
I'm wondering why theres no cache to this tab so that i might see old data
Also, if the feed had failed to load why should i lose access to trending repositories and awesome lists

![Screenshot_20230822_180407_GitHub](https://github.com/community/community/assets/836551/6b57e187-81cc-4439-a4d1-4ea67657dc26)
![Screenshot_20230822_180428_GitHub](https://github.com/community/community/assets/836551/e17824e7-6711-4df8-b6cb-9bf0a1941d2d)

**Comment by alcere** [comment]
Hey @elibyy, thanks for reaching out! I agree, this is not great and we are sorry! 
Do you see this kind of error only visiting the Explore view?

**Reply by elibyy** [reply]
Yeah. This only occurrs in the explore view and only on degraded network, e.g. cellular on the go

**Reply by alcere** [reply]
Thanks for the reply! I can see how this is an inconvenience! I will open an internal issue to explore possible solutions and offer a smoother experience in the future. Thanks for reporting!

**Comment by nasrumikail** [comment]
Good work for me @elibyy @alcere @zackfern

**Comment by nasrumikail** [comment]
Good work for me @elibyy @alcere @zackfern ```

```

**Comment by nasrumikail** [comment]
I'm Ethiopian person
Goota dhaalamaa deemu kumaatamaan hiriirseen biyya too badhaadhina itti fufsiisa@nasrumikail @samus-aran @timoyates @Nishnha @omerbensaadon @tanmayeekamath # # @otter-computer @ghostinhershell

**Reply by nasrumikail** [reply]
@nasrumikail @alcere @elibyy

**Reply by nasrumikail** [reply]
- [ ] @nasrumikail @alcere @elibyy

**Comment by Franklyn-R-Silva** [comment]
Hello,

It sounds like you're enjoying exploring and using a mobile app, but you're encountering some issues when you're on the go with limited data connectivity. You're curious why the app doesn't have a cache feature to store older data and why you lose access to trending repositories and awesome lists when the feed fails to load. These are valid points, and I can provide some insights into possible reasons:

1. **Data Storage and Privacy**: One reason some apps might not have a robust caching system is to prioritize user privacy and data storage. Storing a lot of cached data can consume storage space on users' devices and potentially lead to privacy concerns if sensitive information is cached without proper encryption.

2. **Freshness of Data**: Many apps, especially those that deal with dynamic content like feeds and trending lists, aim to provide users with up-to-date information. Caching data for a long time can lead to users seeing outdated or irrelevant information, which might negatively impact their experience.

3. **Data Consistency**: If the app relies on real-time updates or dynamic content, using a cache might lead to data inconsistencies. For instance, if a trending repository suddenly gains a lot of attention, it might not appear in the cache until it's refreshed, potentially causing confusion for users.

4. **Offline Experience**: Some apps are designed to provide a seamless offline experience, while others prioritize real-time connectivity. Depending on the app's design and goals, the developers might not have focused on creating an extensive cache system.

5. **Technical Challenges**: Implementing a cache system that works smoothly with dynamic content, handles synchronization, and deals with potential conflicts can be quite complex. Developers might prioritize other features and improvements over implementing a cache system.

6. **User Experience**: While it's frustrating to lose access to trending repositories and awesome lists when the feed fails to load, it might be a trade-off to maintain a consistent and user-friendly experience. Displaying cached content might not provide the best experience if it's not guaranteed to be accurate or up-to-date.

If you find these limitations hindering your experience, consider providing feedback to the app's developers or support team. They might appreciate hearing from users like you and could potentially consider adding cache features or improvements to enhance the user experience while balancing the challenges mentioned above.
